spec/features/proposals/index_spec.rb

```diff
@@ -20,20 +20,72 @@
     user = create(:user)
     proposals = create_list(:proposal, 2, observer: user)
     cancelled = create_list(:proposal, 2, status: "cancelled", observer: user)
+    @page = MyRequestsPage.new
 
     login_as(user)
-    visit proposals_path
+    @page.load
+
+    expect(@page.pending.desc_column_header).to have_content "Submitted"
+    expect(@page.cancelled.desc_column_header).to have_content "Submitted"
+
+    expect_order(@page.pending, proposals.reverse)
+    expect_order(@page.cancelled, cancelled.reverse)
+  end
+
+  describe "status field text" do
+    context "when the user is an approver or purchaser" do
```

Comment:
what about one context for approver and one for purchaser? (2 expectations per spec)